test/BUILD.gn

```diff
@@ -1102,6 +1103,14 @@ test("brave_browser_tests") {
     ]
   }
 
+  if (enable_request_otr) {
```

Comment:
this should go in `//brave/browser/request_otr/BUILD.gn` and just have a dep here for `//brave/browser/request_otr:browser_tests`

Reply:
fixed